doc/container.md

```diff
@@ -123,6 +124,139 @@ the following sections for how to add more interfaces and manage your
+        dir:/media/usb/myimage:1.1   -- Use myimage v1.1 from USB media
+        docker-archive:/tmp/archive  -- Use archive:latest from tarball
+        oci-archive:/lib/oci/archive -- Use archive:latest from OCI archive
+                                        May be in .tar or .tar.gz format
+
+        Note: if a remote repository cannot be reached, the creation of the
+              container will be put on a queue that retries pull every time
+              there is a route change in the host's system.
+
+> **Note::** the built-in help system in the CLI is generated from the
+> YANG model, so the same information is also available for remote
+> NETCONF users.
+
+The two most common variants are `docker://` and `oci-archive:/`.
+
+The former requires a working Docker registry and the latter operates on
+a plain OCI archive.  Infix does not come with a built-in registry, so
+the `docker://` option is best used with external services, which in
+turn require [networking][0] to be up.  In a deployment phase the
+easiest may be to set up a single interface on your host system with
+DHCP client.
+
+The default method is `docker://`, so when setting the `image` for your
+container, you can omit the `docker://` prefix.  You can also use the
+admin-exec command `container pull docker://...`, and when configuring a
+container `podman` will check first if it has the image before trying to
+download anything.  (See also the upgrade section, below.)
+
+The `oci-archive:/` is interesting since many users may not have, or do
+not want to, publish their images in a registry.  Use the Docker [OCI
+exporter][5] or any other tool that supports generating [OCI Image][3]
+format.  Infix supports loading both `.tar` or `.tar.gz` formats.
+
+Here we show a simple example of fetching an OCI image to the system,
+but many others exist, tools like `wget`, `curl`, and `scp` come to
+mind.
+
+**Shell OCI Example:**
+
+	admin@example-c0-ff-ee:~$ cd /var/tmp/
+	admin@example-c0-ff-ee:/var/tmp$ sudo wget https://github.com/kernelkit/curiOS/releases/download/edge/curios-oci-amd64.tar.gzConnecting to github.com (140.82.121.3:443)
+	wget: note: TLS certificate validation not implemented
+	Connecting to objects.githubusercontent.com (185.199.109.133:443)
+	saving to 'curios-oci-amd64.tar.gz'
+	curios-oci-amd64.tar 100% |*********************************| 7091k  0:00:00 ETA
+	'curios-oci-amd64.tar.gz' saved
+	admin@example-c0-ff-ee:/var/tmp$ ll
+	total 7104
+	drwxr-xr-x    3 root     root          4096 Mar 27 14:22 ./
+	drwxr-xr-x   14 root     root          4096 Mar 27 11:57 ../
+	-rw-r--r--    1 root     root       7261785 Mar 27 14:22 curios-oci-amd64.tar.gz
+	drwx------    6 frr      frr           4096 Mar 27 11:57 frr/
+
+Importing the image into podman can be done either from the CLI
+admin-exec context ...
+
+    admin@example-c0-ff-ee:/var/tmp$ cli
+    admin@example-c0-ff-ee:/> container load /var/tmp/curios-oci-amd64.tar.gz name curios:edge
```

Comment:
Not necessarily something to now, but I think we should settle on a standard of how we specify local paths. IIRC, the copy command requires a `file://` URI to copy a local file.

Reply:
I've tried to rid myself of $OLDJOB habits, so there's no `file://` for local files in the CLI `copy` command, and that's what I continued with here.

Ever since we switched the default shell back to Bash this has become much clearer to me.  I think we should just allow plain UNIX (absolute) paths.  The `file://` syntax has its place when YANG models or other standards require a proper [URI syntax](https://datatracker.ietf.org/doc/html/rfc3986#section-1.1), but I don't see a reason for us to follow that in Infix CLI commands.

Reply:
Agreed -- UNIX for the win! 

But ...
```
root@infix:~$ ll /cfg/
total 68
drwxr-xr-x    1 root     root          4096 Mar 28  2024 ./
drwxr-xr-x   23 root     root           344 Jan 17  2024 ../
-rw-r--r--    1 root     root         13990 Mar 28  2024 backup-config.cfg
drwxr-xr-x    4 root     root          4096 Nov 14  2103 ssl/
-rw-r--r--    1 root     root         13926 Mar 28  2024 startup-config.cfg
-rw-r--r--    1 root     root          9434 Mar 27  2024 upgrade-config.cfg
-rw-r--r--    1 root     root         13990 Mar 28  2024 wkz.cfg
root@infix:~$ cli

See the 'help' command for an introduction to the system

root@infix:/> copy /cfg/wkz.cfg running-config
Error: invalid source argument: /cfg/wkz.cfg
```
... what am I doing wrong? :smile:

Reply:
Well ... even with the best of intentions, seems I screwed up that one! :sweat_smile: 

On it ... permission and ownership fixes are on another branch though.

Reply:
Fixed, works now in upcoming push!